src/pages/menu/components/menuOption/index.tsx

```diff
@@ -2,21 +2,20 @@
 import { css } from "@emotion/react";
 import { Delete, Edit } from "common/components/icons";
 import useClickOutside from "common/hooks/useClickOutside";
-import { HTMLAttributes, useState } from "react";
+import { useState } from "react";
 import colors from "styles/color";
 import fonts from "styles/font";
+import type { MenuOptionProps } from "./index.types";
 
-interface MenuProps extends HTMLAttributes<HTMLDivElement> {
-  title: string;
-  options: { name: string; price: number }[];
-}
-
-const MenuOption = ({ title, options, ...rest }: MenuProps) => {
+const MenuOption = ({ option, onEdit, onDelete, ...rest }: MenuOptionProps) => {
   const [isSelected, setIsSelected] = useState<boolean>(false);
   const { ref } = useClickOutside<HTMLDivElement>(() => setIsSelected(false));
+  const selectOptionStatus = `${option.isRequired ? "필수" : "선택"} - ${option.isMultiple ? "다중 선택" : "단일 선택"}`;
```

Comment:
변수명이 단일값처럼 느껴져서 요런 느낌으로 표현해줘도 괜찮을 것 같슴다~
```suggestion
  const selectOptionStatusText = `${option.isRequired ? "필수" : "선택"} - ${option.isMultiple ? "다중 선택" : "단일 선택"}`;
```

Reply:
@Seung-wan 네! 좋습니당!